canaryconfigmgr/canaryConfigMgr.go

```diff
@@ -266,23 +344,21 @@ func (canaryCfgMgr *canaryConfigMgr) updateHttpTriggerWithRetries(triggerName, t
 	for i := 0; i < fission.MaxRetries; i++ {
 		triggerObj, err := canaryCfgMgr.fissionClient.HTTPTriggers(triggerNamespace).Get(triggerName)
 		if err != nil {
-			log.Printf("Error getting http trigger object : %v", err)
-			return err
+			return errors.Wrap(err, "error getting http trigger object")
 		}
 
 		triggerObj.Spec.FunctionReference.FunctionWeights = fnWeights
 
 		_, err = canaryCfgMgr.fissionClient.HTTPTriggers(triggerNamespace).Update(triggerObj)
 		switch {
 		case err == nil:
-			log.Printf("Updated Http trigger : %s.%s", triggerName, triggerNamespace)
+			canaryCfgMgr.logger.Info("updated http trigger", zap.String("trigger", fmt.Sprintf("%s.%s", triggerName, triggerNamespace)))
```

Comment:
hmm, aren't we supposed to have two zap.Strings, one for each of name and namespace?

Reply:
correct, I'll fix that up